pkg/tsdbctl/query.go

```diff
@@ -152,12 +184,12 @@ func (qc *queryCommandeer) query() error {
 	}
 
 	if err != nil {
-		return errors.Wrap(err, "Select Failed")
+		return errors.Wrap(err, "The query selection failed.")
 	}
 
 	f, err := formatter.NewFormatter(qc.output, nil)
 	if err != nil {
-		return errors.Wrap(err, "failed to start formatter "+qc.output)
+		return errors.Wrap(err, "Failed to start formatter '"+qc.output+"'.")
```

Comment:
use `errors.Wrapf(err, "msg", params)`

Reply:
I don't understand exactly what you want me to change, and as this request is not related to the changes that I made (the code already used `Wrap`, I just edited the string), please make the changes that you want in a separate commit after we merge this PR.

Reply:
np

Reply:
So, I ended up making the change after I merged the `development` branch :-) — see commit 6e3a4dd.